homeassistant/components/buienradar/util.py

```diff
@@ -79,7 +81,9 @@ async def schedule_update(self, minute=1):
         """Schedule an update after minute minutes."""
         _LOGGER.debug("Scheduling next update in %s minutes", minute)
         nxt = dt_util.utcnow() + timedelta(minutes=minute)
-        async_track_point_in_utc_time(self.hass, self.async_update, nxt)
+        self.unsub_schedule_update = async_track_point_in_utc_time(
+            self.hass, self.async_update, nxt
+        )
```

Comment:
Without this, `async_track_point_in_utc_time` doesn't get cancelled on entry unload.